make distclean error

Got the following errors when running VPATH `make distclean`. Main branch is used.
```
make[1]: Leaving directory '/homes/wkliao/MPICH/Github/build/examples'
Making distclean in test
make[1]: Entering directory '/homes/wkliao/MPICH/Github/build/test'
Making distclean in mpi
/bin/sh: line 20: cd: mpi: No such file or directory
make[1]: *** [Makefile:877: distclean-recursive] Error 1
make[1]: Leaving directory '/homes/wkliao/MPICH/Github/build/test'
make: *** [Makefile:30865: distclean-recursive] Error 1
```

--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -6,7 +6,7 @@
 ## FIXME: commented out temporarily, really just want "mpi" to be dealt with at
 ## distclean time for now
 ##SUBDIRS = mpi util basic commands .
-SUBDIRS = mpi commands .
+SUBDIRS = commands .
 
 # Test both the MPI routines and the MPICH command scripts
 testing: mpi/Makefile

